tests/fixtures/issues/issue-900.php

```diff
@@ -16007,59 +15901,6 @@
         'lite' => false,
         'standard' => true,
     ],
-    'issue-900-LF' => [
-        'ua' => 'Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US; rv:1.9.2.3) Gecko/20100401 Mozilla/5.0 (X11; U; Linux i686; it-IT; rv:1.9.0.2) Gecko/2008092313 Ubuntu/9.25 (jaunty) Firefox/3.8',
```

Comment:
We saw this in our logs too, but it's one of two `3.8` patterns. I believe this was moved to "invalid-versions" file.